src/common/subprocess/subprocess.h

```diff
@@ -310,7 +310,7 @@ flux_future_t *flux_subprocess_kill (flux_subprocess_t *p, int signo);
  */
 void flux_subprocess_ref (flux_subprocess_t *p);
 void flux_subprocess_unref (flux_subprocess_t *p);
-void flux_subprocess_destroy (void *arg);
+#define flux_subprocess_destroy(x) flux_subprocess_unref(x)
 
```

Comment:
Seems like you probably don't need to export the #define for flux_subprocess_destroy(). The API should just specify to use `unref`, unless I'm missing some reason the API requires a destroy?

Reply:
It was primarily for convenience, b/c ```destroy``` is the common function name we use for this.

Could go either way.  Do you think it would be cleaner to just remove it outright?